stopify-continuations/src/callcc/jumper.ts

```diff
@@ -84,47 +83,34 @@ function func(path: NodePath<Labeled<FunctionT>>, state: State): void {
   }
   const restoreLocals = path.node.localVars;
 
-  // We instrument every non-flat function to begin with a *restore block*
```

Comment:
Why did you delete this comment? Bring it back.

Reply:
Done.